.changeset/format-changelogs.cjs

```diff
@@ -19,17 +19,29 @@ async function extractInformation(changeset) {
   };
 }
 
-function getDependencyReleaseLine() {
-  return '';
+async function summarize(changeset) {
+  const { contributor, link, summary } = await analyze(changeset);
+
+  const line = [link, summary, contributor].filter(Boolean).join(' ');
+
+  return `- ${line}`;
 }
 
-async function getReleaseLine(changeset) {
+async function getDependencyReleaseLine(changesets) {
```

Comment:
Should packages in `embroider-css-modules` depend on one another, `getDependencyReleaseLine()` helps ensure that dependent packages get changelog entries and a version bump.